### ISSUE TITLE
Alive and Kicking suggestions

### ISSUE THREAD

**Issue by somedoodest** [BEEmod/BEE2-items#3322]
Here are three suggestions based on a video from "Dot Operator" showcasing his work on the mod's music and sounds. Of course, these should be added with permission from the mod's creators.

1. The Clean version of the Unstationary Scaffold should be changed to this version from Alive and Kicking.
![obraz](https://user-images.githubusercontent.com/34423155/84322088-b0dc3900-ab74-11ea-9165-640d84d2a5c3.png)
(Top view)
![obraz](https://user-images.githubusercontent.com/34423155/84322217-e719b880-ab74-11ea-900a-16bee6bd7082.png)

2. Add HEP Music, which should be added automatically via the compiler, like Laser Music.

3. Add music from the mod once it releases. In my opinion, the music is very pleasant!

Here is the video I'm referring to: https://www.youtube.com/watch?v=U9nmN-3sfjU

**Comment by SCIENTIST2004** [comment]
It looks amazing!

**Comment by Luke18033** [comment]
Alive and Kicking hasn't been released yet, so we don't currently have access to these assets. Additionally I'm pretty sure HEP music requires source code access (which they would have gotten).

**Comment by somedoodest** [comment]
Yeah, it hasn't, but once it releases and you get the permission, you should add these.
I'm sure there can be some kind of hacky workaround for HEP music, like an extra entity attached to the pellet if HEP music is enabled.

**Comment by Luke18033** [comment]
It might be possible to replace the pellet's idle sound (which we already have to pack in) with music. The main problem then is that we don't have any music to use for it.

**Comment by somedoodest** [comment]
Well, HEP music is in Alive and Kicking. We'll probably get music to use from that once it releases.

**Comment by somedoodest** [comment]
Here's an extra idea for HEP music: If possible, there should be an Itemvar to make Infinite Pellet's emit lower-pitched versions of the regular HEP music.

**Comment by maxdevgithub** [comment]
> Add music from the mod once it releases. In my opinion, the music is very pleasant!
I found it i think: https://youtu.be/5yzLxq9vek0
